General queries/File footprint.txt

```diff
@@ -10,3 +10,13 @@ project ComputerName, ActionType, FileName, InitiatingProcessFileName, EventTime
 | summarize makeset(ActionType), FirstEventTime=min(EventTime), (LastEventTime, LastActionType)=arg_max(EventTime, ActionType) by FileName, MatchedSide, ComputerName
 | top 1000 by LastEventTime desc
 | sort by ComputerName, LastEventTime desc
+
+// Query # 2 - Shows you a list of distinct IP addresses and DNS names the endpoint had network communication with through a specific file. 
+// Use this list to whitelist/blacklist IP addresses or understand if there are communication with IP you are not aware of.
+// Update the filename to the name you wish to investigate network communication.
+let filename = "FILENAME GOES HERE";
+// Builds table for distinct URLs based off filename
+NetworkCommunicationEvents 
+| where InitiatingProcessFileName =~ filename and ( isnotempty(RemoteIP) or isnotempty(RemoteUrl) )
+| project filename=InitiatingProcessFileName, DNS=RemoteUrl, IP=RemoteIP
```

Comment:
I don't think you need this line. I'm sure you don't need to project filename.

Reply:
Agreed. please remove